### PR TITLE
cleaning up deadcode and adding tests for task detection. 

### PR DESCRIPTION
**Pull request by kensipe**
first test in executor project

Improvements to the taskInfoBuilder as well.

### CODE REVIEW

**Comment by kensipe** [comment]
@elingg @gabrielhartmann I ran across some code to clean up... it lead to this... looking for a review.

File: hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/scheduler/Task.java [hunk 1]
```diff
@@ -31,9 +31,8 @@ public Task(
     String type,
     String idName) {
 
-    this.info = new TaskInfoBuilder(String.format("task.%s.%s", type, idName), name)
+    this.info = new TaskInfoBuilder(String.format("task.%s.%s", type, idName), name, offer.getSlaveId().getValue())
       .setExecutorInfo(execInfo)
```
Comment (elingg) [inline hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/scheduler/Task.java]: I think we still need to set the slaveid. Is this safe to remove?

**Comment by elingg** [comment]
Thanks @kensipe. Took a quick look and it LGTM. It is a small change, but it might be worth testing to make sure we're not missing anything.

**Comment by kensipe** [comment]
tested good.  merging!

**Pull request merged**